CHANGELOG.md

```diff
@@ -3,10 +3,17 @@
 ## unreleased
 
 * [CHANGE] Scraping: Remove implicit fallback to the Prometheus text format in case of invalid/missing Content-Type and fail the scrape instead. Add ability to specify a `fallback_scrape_protocol` in the scrape config. #15136
-* [CHANGE] Remote-write: default enable_http2 to false.
-* [ENHANCEMENT] Scraping, rules: handle targets reappearing, or rules moving group, when out-of-order is enabled. #14710 
-- [BUGFIX] PromQL: Fix stddev+stdvar aggregations to always ignore native histograms. #14941
-- [BUGFIX] PromQL: Fix stddev+stdvar aggregations to treat Infinity consistently. #14941
+* [CHANGE] Remote-write: default enable_http2 to false. #15219
+* [CHANGE] Scraping: normalize "le" and "quantile" label values upon ingestion. #15164
+* [CHANGE] Scraping: config `scrape_classic_histograms` was renamed to `always_scrape_classic_histograms`. #15178
+* [CHANGE] Config: remove expand-external-labels flag, expand external labels env vars by default. #14657
+* [CHANGE] Adopt log/slog and remove go-kit/log. #14906
```

Comment:
I'm not sure if switching log libraries are relevant to Prometheus users.

I know that this could only be done on 3.0 because this had breaking changes to something. I have vague memories that it changes a few flags... is that it @SuperQ @tjhop?

It would be nice if we change the entry here only with what's relevant to the final user

Reply:
Is it included because it affects those using Prometheus as a library?

Reply:
The log format changes too. I mentioned it in the migration guide as well. While perhaps not crucial, seems like it might avoid a headache for a few users while not hurting anyone else?

Reply:
Should we then say that log format is changing and mentioning what are the changes (if the list is not super huge)?

Reply:
Good point, in fact the same was already done for the beta.1 release notes. I removed the duplicate entry.